### PR TITLE
[ui] Popover for partition segments

### PR DESCRIPTION
**Pull request by hellendag**
## Summary & Motivation

Implement a popover to display the list of partitions that are true/false/skipped for a given evaluation policy condition.

https://github.com/dagster-io/dagster/assets/2823852/be235d50-98d6-471f-be74-9eca7d2fa8c2

## How I Tested These Changes

Storybook

### CODE REVIEW

**Comment by hellendag** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/18911?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `master`
  * **PR #18875** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/18875?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #18911** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/18911?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/18911?utm_source=stack-comment).

File: js_modules/dagster-ui/packages/ui-core/.storybook/preview.js [hunk 1]
```diff
@@ -25,6 +26,7 @@ const GlobalStyle = createGlobalStyle`
   }
 
   html, body {
+    color-scheme: ${browserColorScheme()};
```
Comment (hellendag) [inline js_modules/dagster-ui/packages/ui-core/.storybook/preview.js:29]: Fix the `color-scheme` setting in Storybook.

**Review by salazarm — APPROVED** [review]
looks slick

**Comment by hellendag** [comment]
### Merge activity

* **Dec 27, 3:14 PM**: @hellendag started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/18911).

**Pull request merged**